backend/src/graphql/resolver/ContributionResolver.test.ts

```diff
@@ -706,627 +591,1031 @@ describe('ContributionResolver', () => {
+          query: listAllContributions,
+          variables: {
+            currentPage: 1,
+            pageSize: 25,
+            order: 'DESC',
+            statusFilter: [],
+          },
+        })
+        expect(contributionListObject).toMatchObject({
+          contributionCount: 7,
+          contributionList: expect.arrayContaining([
+            expect.not.objectContaining({
+              state: 'DELETED',
             }),
-          ).resolves.toEqual(
             expect.objectContaining({
-              errors: [new GraphQLError('Can not delete contribution of another user')],
+              amount: '100',
+              state: 'CONFIRMED',
+              id: contributionToConfirm.data.createContribution.id,
+              memo: 'Test contribution to confirm',
             }),
-          )
+            expect.objectContaining({
+              id: pendingContribution.data.createContribution.id,
+              state: 'PENDING',
+              memo: 'Test PENDING contribution update',
+              amount: '10',
+            }),
+            expect.objectContaining({
+              id: contributionToDeny.data.createContribution.id,
+              state: 'DENIED',
+              memo: 'Test contribution to deny',
+              amount: '100',
+            }),
+            expect.objectContaining({
+              id: inProgressContribution.data.createContribution.id,
+              state: 'IN_PROGRESS',
+              memo: 'Test IN_PROGRESS contribution',
+              amount: '100',
+            }),
+            expect.objectContaining({
+              id: bibiCreatedContribution.id,
+              state: 'CONFIRMED',
+              memo: 'Herzlich Willkommen bei Gradido!',
+              amount: '1000',
+            }),
+            expect.objectContaining({
+              id: expect.any(Number),
+              state: 'CONFIRMED',
+              memo: 'Whatever contribution',
+              amount: '166',
+            }),
+            expect.objectContaining({
+              id: expect.any(Number),
+              state: 'DENIED',
+              memo: 'Whatever contribution',
+              amount: '166',
+            }),
+          ]),
         })
+      })
 
-        it('logs the error found', () => {
-          expect(logger.error).toBeCalledWith(
-            'Can not delete contribution of another user',
-            expect.any(Object),
-            expect.any(Number),
-          )
+      it('returns all CONFIRMED contributions', async () => {
+        const {
+          data: { listAllContributions: contributionListObject },
+        }: { data: { listAllContributions: ContributionListResult } } = await query({
+          query: listAllContributions,
+          variables: {
+            currentPage: 1,
+            pageSize: 25,
+            order: 'DESC',
+            statusFilter: ['CONFIRMED'],
+          },
+        })
+        expect(contributionListObject).toMatchObject({
```

Comment:
Test the length of the result array as well

Reply:
Added